docs/docs/spec.md

```diff
@@ -328,10 +332,11 @@ Our declaration annotation is specified to be *recognized* when applied to the
 locations listed below:
```

Comment:
I'm noticing that this CL also covers the locations for _declaration_ annotations. That sounds good to me, but you might tweak the PR description (and watch out for what description GitHub uses at merge time).